spec/controllers/curation_concerns/single_use_links_viewer_controller_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'spec_helper'
+
+describe CurationConcerns::SingleUseLinksViewerController do
+  routes { CurationConcerns::Engine.routes }
+  let(:file) do
+    GenericFile.create do |file|
+      file.add_file(File.open(fixture_path + '/world.png'), path: 'content', original_name: 'world.png', mime_type: 'image/png')
```

Comment:
Should this be Hydra::Works::AddFileToGenericFile?  I couldn't find where add_file gets define or comes from.